docs/platforms/bebop.rst

```diff
@@ -25,15 +25,30 @@ Installing libEnsemble and Dependencies
 ---------------------------------------
 
 You should have an indication that the virtual environment is activated.
-Install mpi4py_ and libEnsemble in this environment, making sure to reference
+Start by installing mpi4py_ in this environment, making sure to reference
 the preinstalled Intel MPI compiler. Your prompt should be similar to the
 following block:
 
 .. code-block:: console
 
     (my_env) user@login:~$ CC=mpiicc MPICC=mpiicc pip install mpi4py --no-binary mpi4py
+
+libEnsemble can then be installed via ``pip`` or ``conda``. To install via ``pip``:
+
+.. code-block:: console
+
     (my_env) user@login:~$ pip install libensemble
 
+Installing libEnsemble via ``conda`` may contain platform-specific
```

Comment:
Think I would remove this line. They are compiled for the general architecture but may not have very specific flags (like AVX-512), and not really different than a pip wheel (I think).